pom.xml

```diff
@@ -237,6 +267,11 @@
                 <version>${axiom-impl.version}</version>
             </dependency>
             <dependency>
+                <groupId>org.apache.axis2.wso2</groupId>
```

Comment:
I can move this to the feature repo. Since the existing code uses axis2 + since it's an external feature, we can keep axis2